public/locations-list-map.js

```diff
@@ -654,15 +673,17 @@ function centerMapToBounds(map, bounds, maxZoom) {
     map.setCenter(params.center);
     map.setZoom(params.zoom);
   } else {
+    google.maps.event.addListenerOnce(map, 'zoom_changed', () => {
```

Comment:
This change seems unrelated to the race. Am I incorrect?

Reply:
For some reason, with the delayed map creation, the maxZoom logic broke when using the &state= param. Either getZoom was returning undefined, or the map tiles wouldn't show.